Makefile

```diff
@@ -14,7 +14,7 @@ all: generate fmt test
 .PHONY: generate-in-docker
 generate-in-docker:
 	@echo ">> Compiling assets and generating Kubernetes manifests"
-	$(CONTAINER_CMD) $(MAKE) $(MFLAGS) generate
+	$(CONTAINER_CMD) make $(MFLAGS) generate
```

Comment:
I’m not sure why that change was necessary for you. Can you revert this part? https://www.gnu.org/software/make/manual/html_node/Recursion.html